doc/iterations/2024.11/add_table_widget.adoc

```diff
@@ -0,0 +1,58 @@
+= Add table widget
+
+== Problem
+
+We want to have a table widget in forms.
+
+== Key Result
+
+A developper is able to contribute a `FormDescription` containing a Table widget.
+The user is able to create a form representation containing a table widget.
+The user is able to edit data with the table widget.
+
+== Solution
+
+Sirius-Web provide the API so that the developper is able to provide the `FormDescription` containing the `TableWidget`.
+
+The table widget is part of a form representation.
+
+The table widget is made of lines and columns.
+It also contains a text area to filter the lines.
+A `Line` is associated with a semantic element.
+A `Cell`, which is the intersection between a line and a column, is also associtaed with a semantic element.
+
+
+In this table, it is possible to:
+
+* *Order*: When clicking/clicking again on the column header, order the lines based on alphanumerical/inverse alphanumerical value.
+* *Filter*: When entering a string in the filter area, to filter lines. A line is displayed if any of its cell contains the filtering string be it alpha or numeric.
+* *Resize* columns(not persisted)
+* *Export* the content in a csv format
+* *Select*: When selecting a line or a cell, the selection is changed with the semantic element associated with the line or the cell.
+* *Edit a cell*: When clicking inside the cell, it is possible to change the content. 
+The change is sent to the server when focussing out.
+** drop down list for mono-valued feature
+** list when check button for multi-valued feature
```

Comment:
How these multi-valued features will be display in the table.
Would it be complicated to use the ref widget?